### ISSUE TITLE
Where are the instructions?

### ISSUE THREAD

**Issue by WULFFJ** [joelsernamoreno/EvilCrow-RF#25]
I bought three of these to do some testing on.  I am newer to RF, but not programming.  Everything installed fine.  However, the lack of instructions is horrible.

What is the power jammer parameter for or what should be assigned?

Do all the other functions of this device call for another sketch?  This device gets advertised as an all in one???

Where are any guides on anything 2.4ghz related for the v2?  

How does this integrate with URH?  You have a video of it, but its selecting another device.  Is it another device that is listening to the Evil Crow?

Overall, these directions don't give a good direction of where to start and is rather annoying.

**Comment by Commander-Crash** [comment]
Agree. The web interface can use some explaining. I wish it was compatible with RFCAT but it seems like it is not a good alt for the YS1

**Comment by Crsarmv7l** [comment]
> Agree. The web interface can use some explaining. I wish it was compatible with RFCAT but it seems like it is not a good alt for the YS1

I have been looking for something like rfcat but with less plug/unplug and more stable. Thought it might be rfquack but that has its own issues.

**Comment by joelsernamoreno** [comment]
Hello

I’ve released a new firmware update.
I’m going to close all existing issues. If you encounter any problems with the new firmware, or if any previous issues haven’t been resolved, please open a new issue.

Thank you!
